src/tools/backend/html.ts

```
@@ -2,6 +2,26 @@
+
+export type HtmlBackend = "claude" | "gemini";
+
+const HTML_CONFIG_KEY = "htmlGenerationBackend";
+
+export function getRawHtmlConfig(context?: ToolContext) {
+  return getRawPluginConfig(context, HTML_CONFIG_KEY);
+}
+
+export function normalizeHtmlConfig(
+  config: string | undefined,
+): HtmlBackend {
```

ESLint

[error] Replace `⏎··config:·string·|·undefined,⏎` with `config:·string·|·undefined`

(prettier/prettier)